driver/pace/driver/driver.py

```diff
@@ -322,6 +322,8 @@ def write_for_restart(
         config_dict["initialization"]["type"] = "restart"
         config_dict["initialization"]["config"]["start_time"] = time
         config_dict["initialization"]["config"]["path"] = restart_path
+        if "case" in config_dict["initialization"]["config"].keys():
```

Comment:
Please comment in code  why we use code to fix earlier configuration versions. It's good to keep track of why those weird `del` exist in code.